pkgs/test/lib/src/runner/browser/chrome.dart

```diff
@@ -3,17 +3,21 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:convert';
 import 'dart:io';
 
+import 'package:coverage/coverage.dart';
+import 'package:http/http.dart' as http;
+import 'package:path/path.dart' as p;
 import 'package:pedantic/pedantic.dart';
 import 'package:test_api/src/backend/runtime.dart'; // ignore: implementation_imports
 import 'package:test_core/src/util/io.dart'; // ignore: implementation_imports
+import 'package:webkit_inspection_protocol/webkit_inspection_protocol.dart';
 
 import '../executable_settings.dart';
 import 'browser.dart';
 import 'default_settings.dart';
 
-// TODO(nweiz): move this into its own package?
```

Comment:
We have https://pub.dev/packages/browser_launcher

Reply:
Are we using it? Should we file an issue to migrate to that package?

Reply:
https://github.com/dart-lang/test/issues/1156